lib/mutant/cli.rb

```diff
@@ -155,6 +155,9 @@ def add_filter_options(opts)
       opts.on('--ignore-subject EXPRESSION', 'Ignore subjects that match EXPRESSION as prefix') do |pattern|
         add_matcher(:ignore_expressions, config.expression_parser.(pattern))
       end
+      opts.on('--since REVISION', 'Only select subjects touched since REVISION') do |revision|
```

Comment:
I brought this up because https://git-scm.com/book/en/v2/Git-Internals-Git-References calls these things references.